lib/modulesync.rb

```diff
@@ -78,16 +78,17 @@ def self.hook(options)
   end
 
   def self.manage_file(filename, settings, options)
+    namespace = settings.additional_settings[:namespace]
```

Comment:
I'm not sure this is correct. If a module is overridden then `module_name` extracts it from the module name rather than the `namespace` setting. Wouldn't this break if `namespace/module` is used?

Reply:
I think it does. Can you imagine a test case that would make the code fail?